hyp3_autorift/process.py

```diff
@@ -45,11 +46,27 @@ def get_lc2_stac_json_key(scene_name):
 
 
 def get_lc2_metadata(scene_name):
+    response = requests.get(f'{LC2_SEARCH_URL}/{scene_name}')
+    try:
+        response.raise_for_status()
+        return response.json()
+    except requests.exceptions.HTTPError:
+        if response.status_code != 404:
+            raise
+
     key = get_lc2_stac_json_key(scene_name)
     obj = S3_CLIENT.get_object(Bucket=LANDSAT_BUCKET, Key=key, RequestPayer='requester')
     return json.load(obj['Body'])
 
 
+def get_lc2_path(metadata):
+    band = metadata['assets'].get('B8.TIF')
```

Comment:
It's a little wonky to have the `B8`/`B08` literal in three different places (lines 63, 132, 203). They're all describing slightly different things, and I'm not sure if that's an argument for treating them separately, or if future me is going to is going to mistakenly assume they're more coupled than they actually are. I don't have any specific suggestions, and it's not a blocker, but I'll mull it a little.

Reply:
Yeah agreed, the band option was needlesly complex, and s2 and l8 are just different enough to have to transform them. Now the l8 is going to `pan`, it's even worse.